### ISSUE TITLE
Typescript usage ?

### ISSUE THREAD

**Issue by drygnet** [Hacksore/bluelinky#124]
Trying to use bluelinky with typescript but I can't get the types right.
(my typescript skills are lacking)

import BlueLinky from "bluelinky";

const client: BlueLinky = new BlueLinky(
	{
		username: "example@test.com",
		password: "mySecretPass",
		pin: "0000",
		region: "EU",
		brand: "hyundai"
	}
);

this will be an american vehicle and the region should not be a string
(Should be REGION.EU, but I can't import the ENUM from node_modules)

can someone point me in a direction of how to use bluelinky with typescript

**Comment by Hacksore** [comment]
Types defs incorrect with the refactoring done via `7.0.0`.

![image](https://user-images.githubusercontent.com/996134/112541570-882ed400-8d81-11eb-9e21-7d87c0dbc944.png)

**Comment by Hacksore** [comment]
@neoPix if you have time would you be able to take a look at this?

**Comment by drygnet** [comment]
Low priority for me, I'm mostly happy that it wasn't (just) my  typescript skills, 
this "hack" works for me for now:

```
import type EuropeanVehicle from "bluelinky/src/vehicles/european.vehicle";
const bluelinky = require('bluelinky')

const client = new bluelinky(....
```


I REALLY APPRECIATE THE WORK YOU (ALL) ARE DOING,
thank you so much ! <3

**Comment by Hacksore** [comment]
Fixed with #125, will be available with the next release.
